.pre-commit-config.yaml

```diff
@@ -55,15 +55,19 @@ repos:
       - id: forbid-new-submodules
       - id: mixed-line-ending
       - id: no-commit-to-branch
-        args: ["--branch", "staging"]
+        args: ["--branch", "master"]
```

Comment:
probably time to https://github.com/github/renaming

Reply:
No disagreement, but I'm nervous about what the fallout might be